config/initializers/booleanizer.rb

```diff
@@ -0,0 +1,7 @@
+class String
+  def to_bool
```

Comment:
it might be easier to just require this value to be `'true'` or `'false'` than the monkeypatch `String`, but I am not super opposed